### PR TITLE
Readme edits

### PR DESCRIPTION
**Pull request by kasiapusz**
have a good day, again.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # hello_world
-another test
+
```
Comment (kasiapusz) [inline README.md:2]: i don't like it.

Comment (TheRealPiotrP) [reply README.md:2]: I like it a lot!

Comment (TheRealPiotrP) [reply README.md:2]: @livarcocc, what do you think??

Comment (livarcocc) [reply README.md:2]: I found some bugs.

Comment (livarcocc) [reply README.md:2]: But looks good.

**Pull request merged**